tracer/src/Datadog.Trace/AppSec/Waf/NativeBindings/WafNative.cs

```diff
@@ -217,30 +187,28 @@ internal IntPtr ObjectMap()
             return ptr;
         }
 
-        internal bool ObjectArrayAdd(IntPtr array, IntPtr entry)
-        {
-            return _objectArrayAddField(array, entry);
-        }
+        internal bool ObjectArrayAdd(IntPtr array, IntPtr entry) => _objectArrayAddField(array, entry);
 
         // Setting entryNameLength to 0 will result in the entryName length being re-computed with strlen
-        internal bool ObjectMapAdd(IntPtr map, string entryName, ulong entryNameLength, IntPtr entry)
-        {
-            return
-                Environment.Is64BitProcess ?
+        internal bool ObjectMapAdd(IntPtr map, string entryName, ulong entryNameLength, IntPtr entry) => Environment.Is64BitProcess ?
                     _objectMapAddFieldX64(map, entryName, entryNameLength, entry) :
                     _objectMapAddFieldX86(map, entryName, (uint)entryNameLength, entry);
-        }
 
-        internal void ObjectFree(IntPtr input)
-        {
-            _objectFreeField(input);
-        }
+        internal void ObjectFreePtr(IntPtr input) => _freeObjectield(input);
```

Comment:
Looks like this isn't being used, I think it should be.

Reply:
It is used [here ](https://github.com/DataDog/dd-trace-dotnet/blob/297eca7993445321247d43d875f62b42f3c1c724/tracer/src/Datadog.Trace/AppSec/Waf/Initialization/WafConfigurator.cs#L71) to free the config obj pointer to `DdwafObjectStruct` type Map

Reply:
Sorry, missed that, thanks!